docs/07_how-to/14_ht-front-end-simple.mdx

```diff
@@ -0,0 +1,273 @@
+  handleFormSubmit(e: CustomEvent) {
+    // handle form submit
+  }
+  ...
+}
+```
+
+**new-page.template.ts**
+
+```ts
+...
+<foundation-form 
+  @submit=${(x, ctx) => x.handleFormSubmit(ctx.event)} 
+  :jsonSchema=${x => x.formJsonSchema} 
+  :uischema=${x => x.formUiSchema}
+>
+</foundation-form>
+...
+```
+
+## Adding a modal
+To create a modal component, add the element to your **new-page.template.ts** file. The component, and any content, will be invisible until the modal is opened.
+
+In the **new-page.ts** class, add the property to get a handle on the modal instance.
+
+```
+export class NewPage extends FoundationElement {
+  myModal: Modal;
+}
+```
+
+In the **new-page.template.ts** add the element and a `ref` value so that the class can open it.
+
+In the click handler on the button, call the `show` method on the modal.
+
+```html
+...
+<rapid-button @click=${x => x.myModal.show()}></rapid-button>
+<rapid-modal ${ref('myModal')}>
+  <p>This paragraph will appear in the modal</p>
+</rapid-modal>
+...
+```
+
+When open, the modal is displayed over the top of any existing content in your page. It also creates an invisible overlay that prevents you user from interacting with any content outside the modal while it is active.
+
+## Adding a flyout
+To add a flyout to your application, add the `<rapid-flyout>` element to your template. 
+
+This element accepts a `closed` property value, a boolean for true or false. When the value is **true**, the flyout will be closed, and when the value is **false** the flyout will be open.
+
+By default, the flyout is positioned on the right side of the screen. To configure it to appear on the left, set `position="left"` on the element.
+
+You can insert any content you want within the flyout. 
+
+:::info
+When the user clicks outside the flyout, it closes. Also, you can add code to close the programmatically; set the value bound to the `closed` attribute.
+
+When a flyout is closed, the component emits a **closed** event, which you can listen to.
+:::
+
+Here is an example declaration of a flyout:
+
+```ts
+<rapid-button @click=${x => x.leftFlyoutActive = true}>Open left positioned flyout</rapid-button>
+<rapid-button @click=${x => x.rightFlyoutActive = true}>Open right positioned flyout</rapid-button>
+<rapid-flyout :closed="${x => !x.rightFlyoutActive}" @closed=${() => console.log('right flyout closed')}>
+  <p>This paragraph will appear in the flyout</p>
+</rapid-flyout>
+<rapid-flyout :closed="${x => !x.leftFlyoutActive}" position="left" @closed=${() => console.log('left flyout closed')}>
+  <p>This paragraph will appear in the flyout</p>
+</rapid-flyout>
+```
+
+## Example application
+We have created an [example application](https://github.com/genesislcap/platform-howto/tree/4a140c118916b177c0dd5fd355a5d5ec36f7d33b/howto-uiexamples) to demonstrate the things you can set up on the front end of a Genesis application.
+
+There are multiple UIs, with links to the corresponding code.
+
+### The back end
```

Comment:
Feels like starting with this in the how to is wrong, shouldn't we keep to readme and have this one pager just talk about the UI code? Content length of back end is same as front end